content/cloud/changelog/spin-cloud-v0-7-0.md

```diff
@@ -0,0 +1,28 @@
+title = "Spin Cloud v0.7.0 Updates"
+template = "changelog_item"
+date = "2024-01-25T12:00:00Z"
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+
+---
+
+## Spin Cloud v0.7.0 Updates
+
+Fermyon recently released the `spin cloud` v0.7.0 update. 
+
+
+## Upgrading to Spin Cloud v0.7.0
+
+To upgrade your `spin cloud` plugin, please use the following command:
+
+```bash
+spin plugins upgrade cloud
+```
+
+If you encounter any issues or have questions during the upgrade process, please don't hesitate to reach out to our [Discord #cloud channel](https://www.fermyon.com/discord).
+
+**References**
+
+- [Spin 2.2 blog post](https://www.fermyon.com/blog/index)
```

Comment:
I think we can remove a reference to a Spin release as this will be independent of a Spin release

Reply:
Thanks @kate-goldenring - have removed that in latest push.